src/FluxCalculator.jl

```diff
@@ -379,38 +379,11 @@ end
 
 update_turbulent_fluxes_point!(sim::Interfacer.SurfaceStub, fields::NamedTuple, colidx::Fields.ColumnIndex) = nothing
 
-differentiate_turbulent_fluxes!(::Interfacer.SurfaceModelSimulation, args) = nothing
-
 """
-    differentiate_turbulent_fluxes(sim::Interfacer.SurfaceModelSimulation, thermo_params, input_args, fluxes, δT_sfc = 0.1)
-
-Differentiates the turbulent fluxes in the surface model simulation `sim` with respect to the surface temperature,
-using δT_sfc as the perturbation.
+    differentiate_turbulent_fluxes!(sim::Interfacer.SurfaceModelSimulation, args)
+This function is required for the Eisenman sea ice model. For all other
```

Comment:
I wouldn't be specific about which currently implemented model uses this. Perhaps we can just say that this provides a place holder for differentiating the fluxes with respect to surface temperature for surface energy balance calculations.